Heber/src/main/java/springApp/java/com/example/demo/models/InstrumentoFinancieroModel.java

```diff
@@ -0,0 +1,68 @@
+package springApp.java.com.example.demo.models;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(
+        use = JsonTypeInfo.Id.NAME,
+        include = JsonTypeInfo.As.PROPERTY,
+        property = "tipo",
+        visible = true // Hacer visible el campo 'tipo' durante la deserialización
+)
+@JsonSubTypes({
+        @JsonSubTypes.Type(value = AccionModel.class, name = "ACCION"),
+        @JsonSubTypes.Type(value = BonoModel.class, name = "BONO")
+})
+public abstract class InstrumentoFinancieroModel {
+
+    private Long id;
+    private String nombre;
+    private double precio;
```

Comment:
Más que manejarlo como una json property podriamos dejarlo diretamente como un atributo de la clase, tambien ya contamos con el enum creado para esto y facilita el acceso y la legibilidad tambien:

```suggestion

public abstract class InstrumentoFinancieroModel {

    private Long id;
    private String nombre;
    private double precio;
    private TipoInstrumento tipo;
```